### ISSUE TITLE
Why is toReplace checked using startsWith?

### ISSUE THREAD

**Issue by egoist** [frostney/rollup-plugin-alias#7]
https://github.com/frostney/rollup-plugin-alias/blob/master/src/index.js#L33

Why not use `===`?

**Comment by egoist** [comment]
I think I got this, but there's an edge case, for example:

``` js
alias({
  vue: require.resolve('vue/dist/vue')
})
```

and I have such file:

``` js
import Vue from 'vue'
import VueRouter from 'vue-router'
```

it throws:

``` bash
ENOENT: no such file or directory, open '/path/to/node_modules/vue/dist/vue.js-router.js'
```

**Comment by adriantoine** [comment]
That's not really an edge case, I want to alias `react` to `preact-compat` or `inferno-compat`, [it is easy to do with Webpack or Browserify](https://github.com/developit/preact-compat#usage-with-webpack) but with this plugin, if I try to import `react-router`, I get:

```
ENOENT: no such file or directory, open 'inferno-compat-router'
```

@frostney can we have a way to use `===`?

**Comment by egoist** [comment]
@adriantoine Currently I use `rollup-plugin-replace` as a workaround

**Comment by adriantoine** [comment]
Nice! But for me the `replace` plugin does the same as this plugin and replaces all `react` occurences. I ended up using [`babel-plugin-module-resolver`](https://github.com/tleunen/babel-plugin-module-resolver).

**Comment by frostney** [comment]
This should be fixed with `1.2.1` and #9 being merged in. I tried the example of using Vue and VueRouter and this worked for me with the latest version.

Please let me know if the issue still persists and feel to free to re-open this issue or open another one.
